dxr/plugins/clang/tests/test_menus/test_menus.py

```diff
@@ -283,3 +283,28 @@ def test_override_pure_overridden(self):
                 'pVirtualFunc',
                 {'html': 'Find overridden',
                  'href': '/code/search?q=%2Boverridden%3ADerivedClass%3A%3ApVirtualFunc%28%29'})
+
+    def test_destructor_declaration(self):
+        """Make sure we link the full destructor name and not just the '~' on a
+        destructor declaration, which can be easily confused for one's
+        complement on the result of a function call."""
+        menu_on(self.source_page('BaseTypes.h'),
+                '~Z',
```

Comment:
I notice there are two '~Z' on the page with menus to check, can we test both? (or does this already?)